src/assets/scripts/client/ui/UiController.js

```diff
@@ -142,7 +145,10 @@ class UiController {
     destroy() {
         this._eventBus = null;
         this.tutorialView = null;
+        this.settingsController = null;
+        // TODO: possibly deprecated
         this.ui = {};
+        // TODO: possibly deprecated
         this.ui.scale = INVALID_NUMBER;
```

Comment:
I think these two lines can be removed. @erikquinn

Reply:
I think so? Can we try removing it real quick? If it still zooms properly, then we should be good, and can answer the question rather than asking it

Reply:
Unable to find this being used anywhere. Removed and verified it doesn't break the zooming, so we should be good. Removed and pushed.